packages/api-gateway/src/drivers/http/adapters/geolocation/handlers.js

```diff
@@ -0,0 +1,44 @@
+async function getPlaces(req, reply) {
+  const { lon, lat, radius } = req.query;
+
+  req.log.info("[http-server]: Getting places: ", { lon, lat, radius });
+
+  //const places = await this.geolocationServices.getPlaces({ lon, lat });
+  const places = [
+    {
+      placeId: "4f7df24s43",
+      placeName: "New loft Parque 1",
+      lat: 45.12346,
+      lon: 47.35332,
+    },
+    {
+      placeId: "09d8fwdf2",
+      placeName: "New loft Parque 2",
+      lat: 31.89201,
+      lon: 32.16765,
+    },
+    {
+      placeId: "98asd791d",
+      placeName: "New loft Parque 3",
+      lat: 11.54765,
+      lon: 12.56736,
+    },
+  ];
```

Comment:
@CETHER as we are depending on the geolocation service we should mock that call using a fixture or something like that. Also we shouldn't hardcoded values so if you want to create fake data I would recommend you to use [faker](https://github.com/faker-js/faker)

Reply:
I will mock the service and use faker to create the fake data. Thank you!